src/wizards/tokenSwapDealWizard/stages/termsStage/termClause/custom.css

```diff
@@ -0,0 +1,118 @@
+:root {
+  /* Overrides the border radius setting in the theme. */
+  --ck-border-radius: 10px;
+
+  /* Overrides the default font size in the theme. */
+  --ck-font-size-base: 14px;
+
+  /* Helper variables to avoid duplication in the colors. */
+  --ck-custom-background: hsl(270, 1%, 29%);
+  --ck-custom-foreground: hsl(255, 3%, 18%);
+  --ck-custom-border: hsl(300, 1%, 22%);
+  --ck-custom-white: hsl(0, 0%, 100%);
+
+  /* -- Overrides generic colors. ------------------------------------------------------------- */
+
+  --ck-color-base-foreground: var(--ck-custom-background);
+  --ck-color-focus-border: #a258a7;
```

Comment:
please use DS variables ($Secondary02)
You can find it under
/src/resources/elements/primeDesignSystem (/colors.scss)